### DISCUSSION TITLE
OMI Election - 2022-04-20

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#211] [General]
# OMI Election

Hey everyone we're almost at the 1 year anniversary of OMI! Congrats to everyone who has been involved!

In order for us to continue to grow, we need to expand our leadership team. At our meeting on April 20th at 16:00 UTC we'll kick off our election for the leadership team. OMI members will have 24 hours to approve the leadership team on Discord. We'll ping @ everyone when the vote has opened.

### Leadership Team Responsibilities

The OMI leadership team will be responsible for the day to day operations of the OMI organization. This include but are not limited to:

- Media
  - Social Media
  - Blog Writing
  - Media Relations
  - Production / Streaming
- Finance
  - Open Collective / Budget
  - Grant Writing / Fundraising
- Events
  - Welcoming people into the community
  - Demo Nights
  - Lightning Talks
  - Hackathons
  - Workshops / teaching
- Partnerships / Liaisons
  - Khronos glTF Working Group
  - Virtual World liaisons / ambassadors
  - W3C Working Groups
  - M3
  - WebXR Discord
  - XRSI
  - And more!
- General Meeting Administration
  - Recording meetings
  - Meeting agenda
  - Meeting facilitators
- Organization Administration
  - Calendar / scheduling
  - Progress summaries
  - GitHub / Backlog
- Working Group Administration
  - W3C Chairs
  - Spec Editors
  - Working Group leadership

The leadership team will meet once a week for an hour to plan out the week's responsibilities and ensure our tasks are covered. As a member of the leadership team you should have at least one hour a week available to meet with the rest of the leadership team as well as some time to accomplish the various tasks.

Members of the leadership team should have good standing with the [W3C Community Guidelines](https://www.w3.org/Consortium/cepc/) and should be role models for the rest of the community. They will be responsible for upholding these guidelines and fostering the growth of a respectful and inclusive community.

Leadership team members will help steer the community via the responsibilities listed above. However, all decisions within the community will still be made via a vote by consensus. Leaders will not have any more weight than any other member in the voting process. The leadership team is responsible for acting on behalf of the community and should take in and address feedback regularly.

### Election Process

People who are interested in volunteering for the leadership team should comment below **before** April 20th at 16:00 UTC. You may also nominate other members before that time, but they must respond confirming their intention to volunteer for the leadership team.

The election will be held asynchronously, members of the community will have 24 hours to confirm members to the leadership team. Every OMI member will be able to individually vote to confirm each member. Members will be confirmed via a simple majority vote. The voting will happen in Discord and you will be notified by an @everyone ping on April 20th at 16:00 UTC.

The term for this position will be 1 year. We'll hold another election during the meeting closest to that date. If we continue holding our weekly meetings on Wednesdays, the next election will be held on April 19th 2023.

Leadership team members may at any point choose to resign, but we'd ask that volunteers think about how much time they are able to commit before volunteering.

As with any OMI member, if at any point a leadership team member is deemed to have violated the W3C Community Guidelines, we will follow the [process outlined by the W3C](https://www.w3.org/Consortium/cepc/#Reporting). Repeat violations will result in the removal of that leader from their position.

### Nominations

Nominate yourself or others in the comments below. If you have been nominated, please confirm your intent to volunteer.

Thank you everyone for an incredible first year at OMI and here's to many more to come!

**Comment by mrmetaverse** [comment]
:hand: I volunteer to continue my service.

**Reply by mrmetaverse** [reply]
I would like to continue my service aligning the org and our high level processes. Continuing to serve as a w3c chair, and filling in as a working group chair for newly formed groups until we achieve sustainability. I am interested in kicking off the portals group, and can continue to run my monthly backlog refinement AND monthly retrospective meetings. I am interested in how we collaborate and form partnerships with other orgs, speaking as an agent of our group, and filling in gaps.

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="503" alt="Screen Shot 2022-04-21 at 6 23 12 PM" src="https://user-images.githubusercontent.com/1753624/164578317-9d10e1db-46a9-4f9f-8785-5ad43e09f94a.png">

Abstained:
<img width="501" alt="Screen Shot 2022-04-21 at 6 23 20 PM" src="https://user-images.githubusercontent.com/1753624/164578326-b02f35e3-83e7-4d93-9e17-6547b38a66f3.png">

Against:
None

**Comment by robertlong** [comment]
I'd like to continue to volunteer to serve as GLTF Working Group Chair and W3C Chair. I would like to step back a bit from my responsibilities in other areas wherever possible, but totally willing to fill in gaps where needed. In the outline above I really want to focus on:

- Partnerships / Liaisons
- Working Group Administration

And filling in gaps in the following areas:

- General Meeting Administration
- Organization Administration

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="507" alt="Screen Shot 2022-04-21 at 6 25 34 PM" src="https://user-images.githubusercontent.com/1753624/164578497-0edb049f-8ec1-4f56-b4cd-6b00b4a115cc.png">

Abstained:
<img width="501" alt="Screen Shot 2022-04-21 at 6 25 40 PM" src="https://user-images.githubusercontent.com/1753624/164578503-02487392-9ced-4159-95c4-478a771415ae.png">

Against:
None

**Comment by antpb** [comment]
I'd like to stay active in the GLTF Working Group and am very interested in helping with W3C duties where appropriate. I've had a lot of experience in large open source projects and have always been curious about the standards process. Given my experience over the last several months, I'm very certain it's where I can help and learn the most. Partnerships / Liaisons is another area or opportunity that pairs nicely with these duties where I can help with implementations or relay information to and from groups.

I'd also like to help with blog writing and continue managing the website.

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="501" alt="Screen Shot 2022-04-21 at 6 27 05 PM" src="https://user-images.githubusercontent.com/1753624/164578659-7fb43828-a168-49af-9016-7ae02f46fa60.png">

Abstained:
<img width="497" alt="Screen Shot 2022-04-21 at 6 27 12 PM" src="https://user-images.githubusercontent.com/1753624/164578681-63da2af6-5c1a-4003-acf9-04ec605a077f.png">

Against:
None

**Comment by paulespino** [comment]
@robertlong Sign me up!

I have vast experience on graphic design, photo, branding, apparel design, web2/3 mobile and web development, research, marketing, brand and product positioning, fund raising, proposal and biz plan writing, project management ...

I feel I would be great at any thing that relates to the above scope. I have time to devote some time evenings on weekdays and on weekends.

**Reply by mrmetaverse** [reply]
:) Thanks Paulespino!

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="496" alt="Screen Shot 2022-04-21 at 6 28 19 PM" src="https://user-images.githubusercontent.com/1753624/164578777-e0e429e9-2f29-413e-b8ba-9f5d18c6b51e.png">

Abstained:
<img width="504" alt="Screen Shot 2022-04-21 at 6 28 28 PM" src="https://user-images.githubusercontent.com/1753624/164578789-5cb26014-5ba5-4cb9-abae-72e247b7e74d.png">

Against:
None

**Comment by musicscene** [comment]
I would like to start by committing to one hour a week assisting with media production and streaming. I am actively working with live streaming volumetric technology, focused on live music and events in built environments. I have a background in audio, video, 2d/3d animation, and solution architecture for carrier-scale application.

**Reply by mrmetaverse** [reply]
amazing! We need this in [omigroup/media](https://github.com/omigroup/media)! :rocket: 
cc: @pcsmgnt

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="506" alt="Screen Shot 2022-04-21 at 6 30 06 PM" src="https://user-images.githubusercontent.com/1753624/164578961-5bbbc3f8-bb29-406c-b955-2c64c122e150.png">

Abstained:
<img width="502" alt="Screen Shot 2022-04-21 at 6 30 12 PM" src="https://user-images.githubusercontent.com/1753624/164578975-5dba21c9-c325-448e-8125-4fc7d677f1a9.png">

Against:
None

**Reply by mrmetaverse** [reply]
@musicscene what is your discord username?

**Reply by musicscene** [reply]
@mrmetaverse my discord is mtv969#6969

**Reply by mrmetaverse** [reply]
Thanks I'll get you added to the OMI admin role!

**Comment by fire** [comment]
✋ I volunteer to service in the areas of:

* Partnerships / Liaisons with Khronos glTF Working Group, Godot Engine and with Virtual Worlds.

I also want to expand OMI towards:

* Media creation to teach learnings from Partnerships / Liaisons.

**Reply by mrmetaverse** [reply]
thanks Fire!

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="497" alt="Screen Shot 2022-04-21 at 6 31 59 PM" src="https://user-images.githubusercontent.com/1753624/164579108-03b040fb-8105-4b6f-a9cf-bb1b85df4f01.png">

Abstained:
<img width="502" alt="Screen Shot 2022-04-21 at 6 32 05 PM" src="https://user-images.githubusercontent.com/1753624/164579124-3e82d61a-c6df-4d0e-a446-418911cab553.png">

Against:
None

**Comment by RedEagleP1** [comment]
✋ I volunteer to service in the areas of:

- Welcoming people and creating channels to bring people to OMI
- Calendar / scheduling -- I really think Discord's event system should be front-and-center here and if you agree I would love to help. 
- Discussion events -- I think there is room for a weekly educational event. I used to lead one but for lack of help I shut down for now. Maybe between OMI and P1 we can cover it? 
- Podcasts -- I run one for P1 and I would love to work together if our joint energy can have double the effect. 

Not keen on leading but helping as time is limited. If there are no others helping in areas (such as welcoming) I would like to keep the overhead on the down-low if possible.

**Reply by mrmetaverse** [reply]
Thanks RedEagle!

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="500" alt="Screen Shot 2022-04-21 at 6 33 39 PM" src="https://user-images.githubusercontent.com/1753624/164579239-78dbbc00-6bdc-48a5-9aaa-7abac3f19357.png">

Abstained:
<img width="503" alt="Screen Shot 2022-04-21 at 6 33 49 PM" src="https://user-images.githubusercontent.com/1753624/164579252-abb26476-4d4b-47b1-ac20-1e7de1e2c5bc.png">

Against:
None

**Comment by madjin** [comment]
✋ I volunteer to service in the areas of:

- Partnerships / Liaisons: with M3 and virtual worlds
- Media: We are making content while live testing avatar interop in M3, can share b-roll and cross-promote since we are sharing feedback and notes with working groups like scripting / gltf-extensions
- Workshops / demo nights: Happy to help facilitate if any are related to avatars

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="506" alt="Screen Shot 2022-04-21 at 6 34 46 PM" src="https://user-images.githubusercontent.com/1753624/164579342-193cdb5f-6da0-4e3a-ba3a-ab53f43e78b9.png">

Abstentions:
None

Against:
<img width="504" alt="Screen Shot 2022-04-21 at 6 34 53 PM" src="https://user-images.githubusercontent.com/1753624/164579363-0a320ee3-c217-4cd5-bb36-b809fece3ebe.png">

**Comment by LightLodges** [comment]
Hi this is Evo - I can facilitate, help guide & continue contributing to organizational development and media teams.

+ Helping refine budgets from planning to production
+ Choosing tech tools and workflows for production and integration with other teams
+ Creating live and hybrid streaming events with global industry allies, collabs internally and with others
+ Assisting in partnership development as part of a team effort, interfacing with public sector orgs (gov and NGO)
+ Currently assisting in posting to & managing social channels including planning to production, moving us to a shared editorial calendar and content pool to collectively publish across channels (decentralize & share social publishing functions with media team)

I contribute to metaverse media reports across fields and industry sectors that this community is also welcome to post & build on for future repos/libraries as useful for new audiences.

I'd also like to work with our community to print up stickers, hoodies and shirts for our contributors this year so we can show up as a community together everywhere across digital and physical worlds alike.

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="504" alt="Screen Shot 2022-04-21 at 6 36 54 PM" src="https://user-images.githubusercontent.com/1753624/164579496-3e153cba-c455-485c-8d4f-3899f16e6b48.png">

Abstentions:
None

Against:
<img width="503" alt="Screen Shot 2022-04-21 at 6 37 03 PM" src="https://user-images.githubusercontent.com/1753624/164579513-93a6368c-278f-4132-84d1-ae2b3cc151e9.png">

**Comment by Elirudite** [comment]
Shadow here, I'd like to help out with the following:

Media
- Blog Writing
- (Maybe) Streaming

Events (Design related)
- Demo Nights
- Lightning Talks
- Hackathons
- Workshops/teaching

Partnerships
- XRSI
- (Maybe) Virtual worlds

Working Group leadership
- Design/Research group (will propose the group in the next week or two)

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="505" alt="Screen Shot 2022-04-21 at 6 38 50 PM" src="https://user-images.githubusercontent.com/1753624/164579641-509631ee-fa24-4fb9-a0b0-a5b2f5b6a399.png">

Abstained:
<img width="502" alt="Screen Shot 2022-04-21 at 6 38 59 PM" src="https://user-images.githubusercontent.com/1753624/164579663-e369ec0c-a2de-409d-886a-1542328457f4.png">

Against:
None

**Comment by humbletim** [comment]
✋ I volunteer to service in the areas of:
- Media: Blog Writing (in particular would like to see an OMI monthly recap)
- Events: Hackathons / Workshops / teaching
- Partnerships / Liaisons: Open Source project outreach and feedback looping
- General Meeting Administration: Meeting facilitation
- Organization Administration
- Working Group Administration: Helping document successful subgroup processes

**Reply by robertlong** [reply]
Confirmation Results: Confirmed

For:
<img width="504" alt="Screen Shot 2022-04-21 at 6 40 28 PM" src="https://user-images.githubusercontent.com/1753624/164579813-3e77fe14-239e-4911-aceb-8c081f6324f3.png">

Abstentions:
<img width="505" alt="Screen Shot 2022-04-21 at 6 40 36 PM" src="https://user-images.githubusercontent.com/1753624/164579835-e2b4e558-d36c-458c-b4e0-106e6c1a852a.png">

Against:
None

**Comment by Knar33** [comment]
After being in the OMI discord for a year now I'm ready to take on a more active role in the organization. I would like to volunteer my time for the following:

- Events
  - Welcoming people into the community
  - Demo Nights
  - Lightning Talks
  - Hackathons
  - Workshops / teaching
- Partnerships / Liaisons
  - Virtual World liaisons / ambassadors
- Organization Administration
  - Calendar / scheduling

On top of this I would like to help bring the OMI Metaverse map into existence and create a working group for it.

**Reply by robertlong** [reply]
Confirmation Status: Confirmed

For:
<img width="503" alt="Screen Shot 2022-04-21 at 6 42 06 PM" src="https://user-images.githubusercontent.com/1753624/164579939-7dba1ebf-8434-449c-b365-264f382f0532.png">

Abstained:
<img width="508" alt="Screen Shot 2022-04-21 at 6 42 12 PM" src="https://user-images.githubusercontent.com/1753624/164579960-74b9b8a1-054d-424c-a2e4-3baba99225e6.png">

Against:
None

**Comment by mrmetaverse** [comment]
All new leaders are added to OMi-Champion role on Discord :checkered_flag:
